### PR TITLE
fixed amount changing issue

### PR DESCRIPTION
**Pull request by tejassonar**
image
image

How to test:
1. Go to membership project (Project Id: proj_LOxkf5GYI054Fi0HcEUF3dKu)
2. try to change frequencies
3. the amount shouldn't change on the next step (contact details page)
4. after going back the default options should be selected (frequency: monthly and default payment option)

Note: In Membership project client checks if `costIsMonthly` is true, then multiply the amount with 12 if the the frequency is yearly.

### CODE REVIEW

File: src/Donations/Micros/DonationTypes/FundingDonations.tsx [hunk 1]
```diff
@@ -104,12 +104,18 @@ function FundingDonations({ setopenCurrencyModal }: Props): ReactElement {
               <div
                 key={index}
                 onClick={() => {
-                  setquantity(option.quantity);
+                  setquantity(
+                    paymentSetup.costIsMonthly && frequency == "yearly"
```
Comment (norbertschuler) [inline src/Donations/Micros/DonationTypes/FundingDonations.tsx:108]: Do you know an example project where costIsMonthly is set? If this value is not set our old logic seems to work fine.

Comment (norbertschuler) [reply src/Donations/Micros/DonationTypes/FundingDonations.tsx:108]: Ok, found one myself: https://donate-with-planet-jfvtupyne-planetapp.vercel.app/?to=proj_LOxkf5GYI054Fi0HcEUF3dKu

**Review by norbertschuler — DISMISSED** [review]
1. Whoever requested this feature with an `costIsMonthly` has not done our project a big favor as it again rising the complexity of the product.
2. The following does not yet work: 

- Choosing "yearly" with e.g. 120€ and going to contact details.
- Then using back arrow to go back to the first screen where then still the "yearly" option is chosen while the chosen amount got reset to 5€ (default option for monthly donation).
- Without any change click the donate button again to go to contact details where you now see "€5.00 every year"

Here as a photo story:


So if we reset the chosen option on the first screen, we also must reset the chosen interval option ("yearly", "monthly" et al) back to the default!

**Comment by ankitecd** [comment]
After selecting Yearly default amount is not selected

Screenshot 2022-01-20 at 18 32 47

**Review by ankitecd — APPROVED** [review]
LGFM

**Pull request merged**